rosdep/python.yaml

```diff
@@ -6515,6 +6515,22 @@ python3-flask-cors:
   ubuntu:
     '*': [python3-flask-cors]
     bionic: null
+python3-flask-jwt-extended:
+  debian:
+    bullseye: [python3-python-flask-jwt-extended]
+    pip:
+      packages: [flask-jwt-extended]
+    sid: [python3-python-flask-jwt-extended]
+  freebsd: [py-flask-jwt-extended]
+  nixos: [python39Packages.flask-jwt-extended]
+  opensuse: [python-flask-jwt-extended]
+  ubuntu:
+    focal: [python3-python-flask-jwt-extended]
+    hirsuite: [python3-python-flask-jwt-extended]
+    impish: [python3-python-flask-jwt-extended]
+    jammy: [python3-python-flask-jwt-extended]
+    pip:
```

Comment:
This pip key currently will be interpreted as a version of ubuntu and never match. This entry should manually call out all older supported platforms on which the pip key should by used to resolve instead of the wildcard. For this it's only `bionic`.

Reply:
resolved in bf1169b2bc4b36c3c5fb44d670d279a9a62bee5d